onnxruntime/core/framework/onnxruntime_typeinfo.cc

```diff
@@ -279,7 +269,7 @@ OrtStatus* OrtTypeInfo::FromTypeProto(const ONNX_NAMESPACE::TypeProto* input, Or
       type_info->denotation = input->denotation();
       *out = type_info;
       return nullptr;
-    }
+    } break;
     case on::TypeProto::kSequenceType: {
```

Comment:
What's the reason for adding the 'break' when there's a 'return' statement immediately prior to it?

Reply:
I don't know. I just reverted minimal build PR

---
In reply to: [508164862](https://github.com/microsoft/onnxruntime/pull/5540#discussion_r508164862) [](ancestors = 508164862)